B/update n rewards rendering

specialist0x opened the pull request.

specialist0x commented on src/state/stake/stake-constants.ts:
```diff
@@ -112,10 +111,10 @@ const NULL_POOL: StakingTri = {
   inStaging: false,
   noTriRewards: false,
-  doubleRewardToken: dummyToken,
+  earnedNonTriRewards: [],
```
This makes adding pools easier, but if a user has farmed tokens that
1. haven't been claimed
2. rewards are turned off

They'll only know if they look at the farm card

rej156 reviewed (approved):
phat pr, lgtm
assuming it's all defensible code

rej156 commented:
request, could you use yarn commit after git add . next time for cleaner commit messages :P

specialist0x commented:
This is a large PR (sorry):

- Using the new APR structure (https://cdn.trisolaris.io/datav2.json)
  - Removed need for
    - adding individual reward tokens
    - indicating if TRI rewards are enabled
  - Rewards keys like `dualRewardToken` and `noTriReward` are dynamically calculated from the AJAX response

- Updated methods to fetch token logos
- Update UI to support `n` reward tokens
- Updated `stake-constants` to remove superfluous information

specialist0x commented:
I'll be squashing this PR when it's merged in

specialist0x commented:
Left is dev, right is prod:
<img width="1291" alt="image" src="https://user-images.githubusercontent.com/94581898/169888658-a9a96659-0c71-4949-bb42-3b7a36eeb80b.png">
<img width="1364" alt="image" src="https://user-images.githubusercontent.com/94581898/169888698-cba02512-cd8e-42c8-889d-881753ff2c7e.png">

memacius32 commented:
will rebase and merge after testing locally with wnear-bbt farm
